### PR TITLE
docs: redirect duplicate doc

### PR DESCRIPTION
**Pull request by vinckr**
- fix: move oauth login flow description
- fix: redirect old doc and delete

as proposed moves the missing flow steps to the new oauth2 login doc, redirects and deletes the old doc.

closes https://github.com/ory/docs/issues/1389

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/oauth2-oidc/custom-login-consent/flow.mdx [hunk 1]
```diff
@@ -220,6 +220,34 @@ The response contains information about the consent request. The body contains a
...
+   `http://<your-ory-network-project>/oauth2/auth?client_id=...&...`.
+2. Ory OAuth2, if unable to authenticate the user (= no session cookie exists), redirects the user's user agent to the Login
+   Provider URL. The application "sitting" at that URL is implemented by you and typically shows a login user interface ("Please
+   enter your username and password"). The URL the user is redirect to looks like
+   `http://login-service/login?login_challenge=1234...`.
+3. The Login Provider, once the user has logged in, tells Ory OAuth2 some information about who the user is (for example the
+   user's ID) and also that the login attempt was successful. This is done using a REST request which includes another redirect
+   URL like `http://<your-ory-network-project>/oauth2/auth?client_id=...&...&login_verifier=4321`.
+4. The user's user agent follows the redirect and lands back at Ory OAuth2. Next, Ory OAuth2 redirects the user's user agent to
+   the Consent Provider, hosted at - for example - `http://consent-service/consent?consent_challenge=4567...`
```
Comment (aeneasr) [inline docs/oauth2-oidc/custom-login-consent/flow.mdx:235]:
```suggestion
   the Consent Provider, hosted at - for example - `https://example.org/oauth2-screens/consent?consent_challenge=4567...`
```

File: docs/oauth2-oidc/custom-login-consent/flow.mdx [hunk 2]
```diff
@@ -220,6 +220,34 @@ The response contains information about the consent request. The body contains a
...
+5. The Consent Provider shows a user interface which asks the user if they would like to grant the OAuth 2.0 Client the requested
+   permissions ("OAuth 2.0 Scope"). You've probably seen this screen around, which is something like: _"Would you like to grant
+   Facebook Image Backup access to all your private and public images?"_.
+6. The Consent Provider makes another REST request to Ory OAuth2 to let it know which permissions the user authorized, and if the
+   user authorized the request at all. The user can choose to not grant an application any access to his/her personal data. In the
+   response of that REST request, a redirect URL is included like
+   `http://<your-ory-network-project>/oauth2/auth?client_id=...&...&consent_verifier=7654...`.
```
Comment (aeneasr) [inline docs/oauth2-oidc/custom-login-consent/flow.mdx:242]:
```suggestion
   `http://https://{project-slug}.projects.oryapis.com/oauth2/auth?client_id=...&...&consent_verifier=7654...`.
```

**Review by aeneasr — CHANGES_REQUESTED** [review]
HTTP is so 1995 - please always use HTTPS :)

And follow the rules: https://github.com/ory/docs#placeholders-and-dummy-data

**Comment by vinckr** [comment]
AH right, thanks for keeping an eye out for this! Will see to keep our own styleguide more in mind 😁

**Review by aeneasr — DISMISSED** [review]

**Comment by vinckr** [comment]
should be ready now @aeneasr

**Review by aeneasr — APPROVED** [review]

**Pull request merged**